wrappers/rust/icicle-core/src/ntt/mod.rs

```diff
@@ -30,6 +30,17 @@ pub enum NTTDir {
 /// a_4, a_2, a_6, a_1, a_5, a_3, a_7`.
 /// - kRN: inputs are bit-reversed-order and outputs are natural-order.
 /// - kRR: inputs and outputs are bit-reversed-order.
+///
+/// Mixed-Radix NTT: digit-reversal is a generalization of bit-reversal where the latter is a private case with 1b
```

Comment:
private case? partial case probably?

Reply:
typos are the same in CUDA-side doc comment

Reply:
I guess it should be special case

Reply:
I commented about that already but I did mean that digit-reversal is the more general case and bit-reversal is a special (or private) case with 1b digits. Why is that wrong?

Reply:
Oh okay. Just unfamiliar usage of this word for me, assumed it was autocorrect or sth

Reply:
I guess I mistranslate from hebrew but special-case is the one I meant. I will fix.